src/modules/instruction-element/dtos/instruction-element.dto.ts

```diff
@@ -7,7 +9,8 @@ export class InstructionElementDto {
   description: string
   mut: boolean
   accountType: InstructionElementAccountType
-  genericType: string
+  genericType: GenericTypeDto
+  genericTypeAccount: AccountEntity | null
```

Comment:
1. genericTypeAccount - > account ?
2. it is not allowed to use entities inside dto model. Use AccountDto if need